cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/transformers/ConvertedT3xModel.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.powsybl.cgmes.conversion.elements.transformers;
+
+import com.powsybl.cgmes.conversion.Context;
+import com.powsybl.cgmes.conversion.elements.transformers.InterpretedT3xModel.InterpretedWinding;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public class ConvertedT3xModel {
+
+    final ConvertedWinding winding1;
+    final ConvertedWinding winding2;
+    final ConvertedWinding winding3;
+    final double ratedU0;
+
+    public ConvertedT3xModel(InterpretedT3xModel interpretedT3xModel, Context context) {
+
+        TapChangerConversion tcc = new TapChangerConversion(context);
+
+        double ratedU0 = interpretedT3xModel.ratedU0;
```

Comment:
Why do we have this line and not directly:
```java
this.ratedU0 = interpretedT3xModel.ratedU0;
```
?

Reply:
Done